airbyte-integrations/connectors/source-posthog/source_posthog/schemas/projects.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
how did we decide which missing streams to add? It's not clear to me why `projects` is added, but not `groups`

Reply:
'projects' is parent stream for all another streams. 

All API URLs like: https://app.posthog.com/api/projects/2331/.../